Initium-Odp/war/odp/javascript/script.js

```diff
@@ -1481,23 +1483,28 @@ $(document).keyup(function(event){
 		var buttonToPress = $('[shortcut="'+event.which+'"]');
 		if (buttonToPress.length>0)
 		{
-			window.disableShortcuts = true;
+			if (buttonToPress[0].hasClass("main-button-half")==false)
+				window.disableShortcuts = true;
```

Comment:
Yeah, it's not really used much, except to disable things for the long operations

Also, added a small fix for the use of quietly in refresh.